config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+# production:
+#   adapter: redis
+#   url: redis://localhost:6379/1
```

Comment:
We don't use Action Cable, I commented out adapter configuration for production, just in case.